src/axis.h

```diff
@@ -22,7 +22,7 @@ class Axis : public QObject {
     Q_OBJECT
 
     //each axis can create a key press or move the mouse in one of four directions.
-    enum Interpretation { ZeroOne, Gradient, Absolute };
+    enum Interpretation { ZeroOne, Gradient, Absolute, Absolute2 = Absolute };
```

Comment:
What is this `Absolute2` good for?

Reply:
there is a comment where I use it
there was a conflict with some external headers, so the name 'Absolute' is reserved